shared/chat/conversation-filter-input.js

```diff
@@ -2,6 +2,9 @@
 import * as React from 'react'
 import * as Kb from '../common-adapters'
 import * as Styles from '../styles'
+import * as Platforms from '../constants/platform'
+
+const platformShortcutSymbol = Platforms.isDarwin ? '⌘' : '^'
```

Comment:
Where should I put this? We'll be using it more often moving forward to teach people about keyboard shortcuts.

Reply:
`constants/platform` seems like a good place